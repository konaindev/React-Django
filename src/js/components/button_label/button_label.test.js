import renderer from "react-test-renderer";
import React from "react";

import ButtonLabel from "./index";

describe("ButtonLabel", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<ButtonLabel label="beta">insights</ButtonLabel>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
