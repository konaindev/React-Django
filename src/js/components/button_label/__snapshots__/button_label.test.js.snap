// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonLabel renders correctly 1`] = `
<div
  className="button-label"
  onClick={[Function]}
>
  <div
    className="button-label__text"
  >
    insights
  </div>
  <div
    className="button-label__label"
  >
    beta
  </div>
</div>
`;
